lib/notification-issue.coffee

```diff
@@ -52,7 +54,10 @@ class NotificationIssue
       "#{repoUrl}/issues/new?title=#{@encodeURI(@getIssueTitle())}&body=#{@encodeURI(issueBody)}"
 
   getIssueTitle: ->
-    @notification.getMessage()
+    title = @notification.getMessage()
+    if title.length > TRUNCATE_TITLE_TO
```

Comment:
Ah ... good point. That's what I get for code reviewing before I've had my breakfast :laughing: